app/src/main/res/layout/fragment_keyboard_2_stroke_ff.xml

```diff
@@ -0,0 +1,255 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why are we keeping this?

Reply:
We're keeping this layout around in case we go back to a two-stroke layout